lib/ffmpeg/movie.rb

```diff
@@ -141,6 +143,10 @@ def initialize(path)
       end
     end
 
+    def ffprobe_command(binary = FFMPEG.ffprobe_binary)
```

Comment:
I think I'd err on the side of caution and create a `def ffmpeg_command` to ensure that it gets called with `@movie.ffprobe_command` instead of `@movie.ffprobe_command(FFMPEG.ffmpeg_binary)` which feels a bit funky, if you know what I mean?
Just a basic override that's along the lines of:
```rb
def ffmpeg_command
  @movie.ffprobe_command(FFMPEG.ffmpeg_binary)
end
```
As silly as it sounds, I would also be inclined to suggest using `ffmpeg_command` as the "base" command, just because of the nature of the ffmpeg foundation and such (people think of ffmpeg, not ffprobe) 😛 

Either that, or extracting it one step further into `ff_command(binary)`, and then calling those via the wrapper's `ffmpeg_command` and `ffprobe_command` so neither get a binary override?